mu4/playback/internal/playbackcontroller.cpp

```diff
@@ -90,7 +114,7 @@ void PlaybackController::playElementOnClick(const notation::Element* e)
 
     LOGD() << midiData.dump(true);
 
-    audioPlayer()->playMidi(midiData);
+    sequencer()->playMidi(midiData);
```

Comment:
I know, what code in the `playMidi` dangerous - not thread-safe.
So, I added 
```
    for (int i = 0; i < 10; ++i) {
        sequencer()->playMidi(midiData);
    }
```
And stable crashe!
Because, Synthesizer not thread-safe, but it `handleEvent` called from different threads

```
11:19:25.182 | INFO  | 140734145468160 | FluidSynth | handleEvent: th: 140734145468160
11:19:25.182 | INFO  | 140733730256640 | FluidSynth | handleEvent: th: 140733730256640
11:19:25.182 | INFO  | 140734145468160 | FluidSynth | handleEvent: th: 140734145468160
11:19:25.182 | INFO  | 140733730256640 | FluidSynth | handleEvent: th: 140733730256640
11:19:25.182 | INFO  | 140734145468160 | FluidSynth | handleEvent: th: 140734145468160
11:19:25.182 | INFO  | 140733730256640 | FluidSynth | handleEvent: th: 140733730256640
11:19:25.182 | INFO  | 140733730256640 | FluidSynth | handleEvent: th: 140733730256640
11:19:25.182 | INFO  | 140733721863936 | FluidSynth | handleEvent: th: 140733721863936
11:19:25.182 | INFO  | 140733721863936 | FluidSynth | handleEvent: th: 140733721863936
11:19:25.182 | INFO  | 140733721863936 | FluidSynth | handleEvent: th: 140733721863936
11:19:25.182 | INFO  | 140733721863936 | FluidSynth | handleEvent: th: 140733721863936
11:19:25.182 | INFO  | 140733721863936 | FluidSynth | handleEvent: th: 140733721863936
11:19:25.182 | INFO  | 140733721863936 | FluidSynth | handleEvent: th: 140733721863936
11:19:25.182 | INFO  | 140733721863936 | FluidSynth | handleEvent: th: 140733721863936
11:19:25.182 | INFO  | 140733721863936 | FluidSynth | handleEvent: th: 140733721863936
11:19:25.182 | INFO  | 140733721863936 | FluidSynth | handleEvent: th: 140733721863936
```

![image](https://user-images.githubusercontent.com/3818029/99873863-a629d380-2beb-11eb-878a-b198d2d39837.png)

Reply:
fixed